src/services/DfusionService.ts

```diff
@@ -0,0 +1,234 @@
+import Web3 from 'web3'
+
+import Logger from 'helpers/Logger'
+import { ContractEventLog } from 'contracts/types'
+import { OrderPlacement, StablecoinConverter } from 'contracts/StablecoinConverter'
+import packageJson from '../../package.json'
+import { BigNumber } from 'bignumber.js'
```

Comment:
Is this a different library than `BN.js` we use on `dex-react`?

Reply:
It is, I want to initiate a conversation about this two libraries to see pros/cons and when do we use each one.

BN is only for positive numbers, I wanted to calculate the price here so I used this one. Happy to review this, specially in the new `dex-js` project

Reply:
This topics will be moved to the dex-js creation: https://github.com/gnosis/dex-telegram/issues/30